### PR TITLE
adds a translation of definition for package (Japanese)

### PR DESCRIPTION
**Pull request by TomKellyGenetics**
## Author: 

- @TomKellyGenetics 

## Language: 

- Japanese

## Terms defined:

- package (パッケージ)

### CODE REVIEW

File: glossary.yml [hunk 1]
```diff
@@ -4188,7 +4188,11 @@
     def: >
       A collection of code, data, and documentation that can be distributed and
       re-used. Also referred to in some languages as a [library](#library) or [module](#module).
-
+  ja:
+    term: パッケージ
+    def: >
+      持つ関数のまとまりとデーターと文書化が分散して、再使えるように環境に加えること出来のこと。
+      それに、何かの言語の場合は「[ライブラリー](#library)」と「[モジュール](#module)」と読んでいのこと。
```
Comment (masamiy) [inline glossary.yml:4195]: Suggestion: 「[ライブラリー](#library)」もしくは「[モジュール](#module)」と呼ぶプログラミング言語もある。

File: glossary.yml [hunk 2]
```diff
@@ -4188,7 +4188,11 @@
     def: >
       A collection of code, data, and documentation that can be distributed and
       re-used. Also referred to in some languages as a [library](#library) or [module](#module).
-
+  ja:
+    term: パッケージ
+    def: >
+      持つ関数のまとまりとデーターと文書化が分散して、再使えるように環境に加えること出来のこと。
```
Comment (masamiy) [inline glossary.yml:4194]: Suggestion: 配布や再利用が可能なコード、データおよびドキュメントの集合体。

**Review by masamiy — CHANGES_REQUESTED** [review]
Thanks heaps, @TomKellyGenetics ! You are truly awesome !!! Please see my suggestions.

File: glossary.yml (hunk 1 above)
Comment (TomKellyGenetics) [reply glossary.yml:4195]: Thanks for the feedback. I think this better reflects the meaning of the English version. The terms aren't exclusive, different languages use "package", "library", "module", "plug-in", "add-on", "extension", etc.

```suggestion
      それに、プログラミング言語のパッケージも「[ライブラリー](#library)」や「[モジュール](#module)」と呼ぶもある。
```

File: glossary.yml (hunk 2 above)
Comment (TomKellyGenetics) [reply glossary.yml:4194]: This seems good to me. I've changed [Documentation](https://en.wikipedia.org/wiki/Documentation#Documentation_in_computer_science) to reflect the specific use in software field. Perhaps the Kanji wasn't clear for this usage.

```suggestion
      配布や再利用が可能なコード、データおよびソフトウェアドキュメンテーション集合体。
```

File: glossary.yml [hunk 3]
```diff
@@ -4188,7 +4188,11 @@
     def: >
       A collection of code, data, and documentation that can be distributed and
       re-used. Also referred to in some languages as a [library](#library) or [module](#module).
-
+  ja:
+    term: パッケージ
+    def: >
+      配布や再利用が可能なコード、データおよびソフトウェアドキュメンテーション集合体。
+      それに、プログラミング言語のパッケージも「[ライブラリー](#library)」や「[モジュール](#module)」と呼ぶもある。
```
Comment (naoe-tatara) [inline glossary.yml:4195]:
```suggestion
      プログラミング言語によっては、「[ライブラリー](#library)」や「[モジュール](#module)」と呼ぶこともある。
```

**Review by naoe-tatara — CHANGES_REQUESTED** [review]
I would suggest a small change in the second line :)

File: glossary.yml (hunk 3 above)
Comment (TomKellyGenetics) [reply glossary.yml]: Thanks I think this works well. 🙂

**Comment by TomKellyGenetics** [comment]
> Thanks I think this works well. 🙂
> 
> _Originally posted by @TomKellyGenetics in https://github.com/carpentries/glosario/pull/299#discussion_r614033508_

Thanks for your help @masamiy and @naoe-tatara. The Japanese translation seems ready to merge.

File: glossary.yml (hunk 2 above)
Comment (masamiy) [reply glossary.yml]: I understand that documentation means documentation, e.g., README? I 'feel' 文書 is okay, but to make an emphasis on it is a document related to code and/or data, what about:
配布や再利用が可能なコード、データおよびそれらに付随する文書をひとまとまりにしたもの。
(Heavily plagiarised wiki -  https://ja.wikipedia.org/wiki/%E3%83%A9%E3%82%A4%E3%83%96%E3%83%A9%E3%83%AA)

**Review by naoe-tatara — APPROVED** [review]
Thanks @TomKellyGenetics !

File: glossary.yml [hunk 4]
```diff
@@ -4191,8 +4191,8 @@
   ja:
     term: パッケージ
     def: >
-      持つ関数のまとまりとデーターと文書化が分散して、再使えるように環境に加えること出来のこと。
-      それに、何かの言語の場合は「[ライブラリー](#library)」と「[モジュール](#module)」と読んでいのこと。
+      配布や再利用が可能なコード、データおよびソフトウェアドキュメンテーション集合体。
```
Comment (masamiy) [inline glossary.yml:4194]: Suggestion: 
配布や再利用が可能なコード、データおよびそれらに付随する文書をひとまとまりにしたもの。
or
 配布や再利用が可能なコード、データおよびそれらに付随する文書の集合体。

File: glossary.yml [hunk 5]
```diff
@@ -4191,8 +4191,8 @@
...
+      それに、プログラミング言語のパッケージも「[ライブラリー](#library)」や「[モジュール](#module)」と呼ぶもある。
```
Comment (masamiy) [inline glossary.yml]: Suggestion: 
[ライブラリー](#library)」や「[モジュール](#module)」と呼ぶプログラミング言語もある。

**Review by masamiy — CHANGES_REQUESTED** [review]
Thank you for the contribution! I thought I did this one... may be in a wrong place? Anyhow, will you please have a look and let me know with your questions :)

**Comment by naoe-tatara** [comment]
Hi @masamiy , I think you see the outdated file. If you go to the "files changed" tab at the top, you will see already revised translations.

**Comment by masamiy** [comment]
Thanks @naoe-tatara for navigating me to a right place. I think the first sentence needs to be changed a little still by at least adding 'の' before '集合体’. Am I looking at a right one??

File: glossary.yml (hunk 4 above)
Comment (TomKellyGenetics) [reply glossary.yml:4194]: As mentioned above, I think ドキュメンテーション(documentation) is better for computer jargon. It seems to be used like this elsewhere in Japanese.

```suggestion
      配布や再利用が可能なコード、データおよびソフトウェアドキュメンテーションの集合体。
```

Comment (TomKellyGenetics) [reply glossary.yml:4194]: To clarify, in this case "[documentation](https://en.wikipedia.org/wiki/Software_documentation)" means manuals and other files to guide users or developers.

File: glossary.yml (hunk 5 above)
Comment (TomKellyGenetics) [reply glossary.yml]: This one's already taken care of in the current version 😃 

```
      プログラミング言語によっては、「[ライブラリー](#library)」や「[モジュール](#module)」と呼ぶこともある。
```

File: glossary.yml (hunk 2 above)
Comment (TomKellyGenetics) [reply glossary.yml]: I still think ドキュメンテーション is more appropriate for this usage for example:
[https://ja.wikipedia.org/wiki/ソフトウェアドキュメンテーション](https://ja.wikipedia.org/wiki/%E3%82%BD%E3%83%95%E3%83%88%E3%82%A6%E3%82%A7%E3%82%A2%E3%83%89%E3%82%AD%E3%83%A5%E3%83%A1%E3%83%B3%E3%83%86%E3%83%BC%E3%82%B7%E3%83%A7%E3%83%B3)
https://en.wikipedia.org/wiki/Software_documentation

**Comment by TomKellyGenetics** [comment]
Thanks @masamiy, I've made grammatical changes as suggested but I think katakana for "documentation" is still more appropriate. https://github.com/carpentries/glosario/pull/299#discussion_r617980899

File: glossary.yml (hunk 2 above)
Comment (naoe-tatara) [reply glossary.yml]: Yes, I agree with @TomKellyGenetics here. ドキュメンテーション is very often used and I think everyone who comes to the Carpentries workshop do understand what this means.

**Review by masamiy — APPROVED** [review]

**Comment by masamiy** [comment]
All done. Thank you for the contribution.

**Pull request merged**